packages/cw1155/src/event.rs

```diff
@@ -1,42 +1,54 @@
-use cosmwasm_std::{attr, Attribute, HumanAddr, Uint128};
+use cosmwasm_std::{HumanAddr, Response, Uint128};
 use cw0::Event;
 
-use crate::msg::TokenId;
-
+/// Tracks token transfer/mint/burn actions
```

Comment:
Interesting that this is all references.
Definitely more efficient, but harder to get write. But nicely done.